pkgs/applications/networking/irc/quassel/default.nix

```diff
@@ -31,7 +32,7 @@ assert !buildClient -> !withKDE; # KDE is used by the client only
 let
   edf = flag: feature: [("-D" + feature + (if flag then "=ON" else "=OFF"))];
 
-in with stdenv; mkDerivation rec {
+in (if daemon && !monolithic then stdenv.mkDerivation else mkDerivation) rec {
```

Comment:
This condition should probably be `if !buildClient then stdenv.mkDerivation else mkDerivation`.